server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/DownloadStepdefs.java

```diff
@@ -95,38 +110,147 @@ public void optionDownload(String username) throws Throwable {
             with.header("Authorization", accessToken.serialize());
         }
 
-        response = with.options("/download/myBlob");
+        response = with
+                .options("/download/" + ONE_ATTACHMENT_EML_ATTACHEMENT_BLOB_ID);
     }
 
     @When("^\"([^\"]*)\" downloads \"([^\"]*)\"$")
     public void downloads(String username, String attachmentId) throws Throwable {
         String blobId = blobIdByAttachmentId.get(attachmentId);
+        response = requestWithAuthentication(blobId, username)
+                .get("/download/" + blobId);
+    }
+
+    private RequestSpecification requestWithAuthentication(String blobId, String username) {
```

Comment:
This method returns an authenticated request ready to use.
It's handling both header & query parameter authentication mechanisms, as we are testing both.